Add TAC meeting files 08-08-2024 (#254)
confidential-computing/governance: TAC/Meetings/2024/2024-08-08/CCC-TAC-Minutes-2024-08-08.md

DM asked to understand the goals and a schedule. MB agreed and stated it was this project was still in the formation period.

## Veraison Project Update

Thomas Fossati and Simon Frost (SF) gave an annual review on [Project Veraison](./Project%20Veraison%20CCC%20Review%202024.pdf).

MB mentioned that he thought the GitHub stats were valuable and would want to see this data whenever we did an annual review.

Discussion ensued.

DM asked about the security badge and then asked which repo would be most appropriate for a new contributor. SF stated that there was a [repo guide](https://github.com/veraison/docs/blob/main/repo-guide.md). DM asked if there had been engagement with the project mentor. Simon stated that there hasn’t. DM stated that there has been discussion about the viability of the liaison model and if it can scale. DM asked if the role of the liaison provides value to the projects.

## Runtime Attestations

Jason Rogers (JR) and Dr. Wesley Peck (WP) presented on [runtime attestations](./Invary%20Runtime%20Integrity%20-%20CCC%20TAC.pdf).

Discussion ensued with interactive questions and answers.

## 2024 CCC TAC Objectives

> Fritz Adler (FA) discussed the 2024 objectives of the TAC. He gave feedback on the academic engagement objective and asked the attendees what academic topics would be of interest to the TAC.

Reviewer: ```suggestion
Fritz Alder (FA) discussed the 2024 objectives of the TAC. He gave feedback on the academic engagement objective and asked the attendees what academic topics would be of interest to the TAC.
```

MB said that talks on mitigations for attacks related to protocols would be valuable.

> FD stated that the PQ talks presented were insightful. He further asked if the group would be interested in academic topics or if the talks should focus on more practical topics.

Reviewer: ```suggestion
FA stated that the PQ talks presented were insightful. He further asked if the group would be interested in academic topics or if the talks should focus on more practical topics.
```

MB said that he would like to see more of the TAC Tech Talks presented at industry events.

> FD said that MB’s point ties in with his next topic, asking the group about what external engagement with academics should be pursued.

Reviewer: ```suggestion
FA said that MB’s point ties in with his next topic, asking the group about what external engagement with academics should be pursued.
```

MB agreed that it would be helpful to build relationships with academics.
Thore Sommer (TS) said that relationships with academics and providing a platform where academic work could be done and technical questions could be asked and answered would be beneficial. MB added that larger organizations with research groups would be helpful.

Ofir Azoulay-Rozanes said that the missing topic for him is the involvement of hardware providers and cloud providers in discussions about vulnerabilities and mitigations, as well as the development of hardware. FA cautioned against the discussion of “soon to be released” features, as it could violate the antitrust policy. DM said that IP announcements that companies are comfortable sharing would be useful information to discuss as a community.

> MB pointed out that the TAC is a technical advisory council, and they should avoid “product pitches.” He suggested seeking advice from the GB about news feeds from vendors, as large companies might overshadow smaller companies. Catherine Zhang (CZ) asked if sponsoring summits would be a good place to discuss academic topics. Discussion ensued about event participation and academic involvement.

Reviewer: ```suggestion
MB pointed out that the TAC is a technical advisory council, and they should avoid “product pitches.” He suggested seeking advice from the GB about news feeds from vendors, as large companies might overshadow smaller companies. Catherine Zhang (CZ) asked if sponsoring academic conferences would be a good opportunity to discuss the CCC with academics. Discussion ensued about event participation and academic involvement.
```
I understood Catherine's comments differently. Feel free to ignore if you disagree.
